ha/core/system_health/system_health_manager.py

```diff
@@ -57,3 +60,21 @@ def key_exists(self, key: str) -> bool:
         if self._store.key_exists(key):
             return True
         return False
+
+    def parse_key(self, lookup, match_with, prefix):
```

Comment:
I think parse_key will misguide others that it just parses but actually it is fetching matched keys.
so suggesting to change it to get_key, fetch_key, find_key or search_key, or something similar.

Reply:
Actually it parses the matching keys and returns value from the key.
Example;
The key will be parsed for lookup.
(lookup='cvg',
match_with={'node_id': 'n01', 'disk':'d01'},
prefix='cortx>ha>cluster>c01')
The below key matches for the above and method returns `cv01`.
`
cortx>ha>cluster>c01>node>n01>cvg>cv01>disk>d01>health
`

Reply:
We don't need key to be returned. Only the parsed value from the key.

Reply:
then you can give a name get_value, find_value, or any such...
bt if you think OK to have current name i.e. also ok